akka-projection-jdbc/src/main/scala/akka/projection/jdbc/javadsl/JdbcHandler.scala

```diff
@@ -23,8 +26,13 @@ object JdbcHandler {
     override def process(session: S, envelope: Envelope): Unit = handler(session, envelope)
   }
 
+  @Deprecated
+  @deprecated("Use the similar method with Java functional interface as a parameter", "1.2.0")
   def fromFunction[Envelope, S <: JdbcSession](handler: (S, Envelope) => Unit): JdbcHandler[Envelope, S] =
     new HandlerFunction(handler)
+
+  def fromFunction[Envelope, S <: JdbcSession](handler: BiConsumer[S, Envelope]): JdbcHandler[Envelope, S] =
+    new HandlerFunction(handler.asScala)
```

Comment:
I'd prefer that we change the above `HandlerFunction` to take a BiConsumer directly instead of using a Scala function.

Reply:
🆗